Common/include/CConfig.hpp

```diff
@@ -8826,6 +8826,14 @@ class CConfig {
    */
   bool GetPrintInlet_InterpolatedData(void) const { return PrintInlet_InterpolatedData; }
 
+  /*!
+   * \brief Get information about using the Reynolds stress tensor and mean rate of strain matrix.
+   * \return <code>TRUE</code> means that they will be used.
+   */
+  bool GetUsing_ReynoldsStress(void) const {
+    return (using_uq);
+  }
```

Comment:
Why is this needed?

Reply:
You are right. Commit 62444fde reverts the proposed changes of the allocation of `CNumerics::MeanReynoldsStress`.